docs/topics/modules/setup_debugger.adoc

```diff
@@ -3,47 +3,54 @@
 
 To debug your project code using the Che workspace Debugger feature:
 
-. In the `Debug` drop down menu, select the `debug` option.
+. Click the `debug` option from the Debug button.
```

Comment:
This doesn't make sense to me. 'debug option from the Debug button'? What is 'an option from a button'? Drop-down menu made more sense to me...

Reply:
Well Run and Debug are slightly tricky UI elements here. Here is how run looks:
![tri_run](https://user-images.githubusercontent.com/16647625/33930601-81470a96-e013-11e7-9d3d-dbf944d4250a.png)
They are similar to Print icons in docs; however you need to click the arrow in the drop-down and then click on Run to make it run. Run is the only option in the drop-down. This is the same with debug on top. Here I tried to keep it consistent with wording  for the Run option mentioned in earlier sections.

Would this make sense: Click the `debug` option from the Debug drop-down.

Reply:
> Would this make sense: Click the debug option from the Debug drop-down.

Yes, better.